### PR TITLE
[BD] Fix pylint import-error

### PR DESCRIPTION
**Pull request by andrey-canon**
## Description 
Remove object unnecessary inheritance.
this https://github.com/edx/edx-analytics-dashboard/pull/985/ is necessary. 
Part of .https://openedx.atlassian.net/browse/BOM-1360

### Reviewers
- [x] @morenol 
- [x]  Is this ready for edX's review? 
- [ ] @jmbowman 
 
### Post-review
- [ ] Rebase and squash commits

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @andrey-canon! I've created [OSPR-4335](https://openedx.atlassian.net/browse/OSPR-4335) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

**Review by morenol — APPROVED** [review]

**Comment by jmbowman** [comment]
Is this ready for review?  Still a draft PR.

File: analytics_dashboard/urls.py [hunk 1]
```diff
@@ -11,9 +11,7 @@
 from django.views import defaults
 from django.views.i18n import JavaScriptCatalog
 
-# pylint suggests importing analytics_dashboard.core, which causes errors in our AMI
```
Comment (jmbowman) [inline analytics_dashboard/urls.py:14]: What's the best way to make sure these changes don't run afoul of the same error?

**Review by jmbowman — COMMENTED** [review]
It looks like Clinton deliberately set things up the way they are to make the imports shorter: https://github.com/edx/edx-analytics-dashboard/blob/master/analytics_dashboard/settings/base.py#L23-L25 .  @macdiesel do you think this is worth changing?

**Comment by estute** [comment]
@jmbowman it looks like Clinton added that from a template when setting up the repo as opposed to some intentional fix specific to this repo. I'm fine with merging this as long we can get some insight into the AMI error comment that you commented on.

**Review by estute — APPROVED** [review]

**Review by jmbowman — APPROVED** [review]
I'm guessing the error in the comment was from having a mix of different ways of doing the imports; not sure there's really a way for eduNEXT to check our AMI for it, so merging for now and we can revert if it causes any issues.

**Pull request merged**